sql-cli/examples/_dags/advanced.py

```diff
@@ -0,0 +1,69 @@
+# NOTE: This is an auto-generated file. Please do not edit this file manually.
+from airflow import DAG
+from airflow.utils import timezone
+from airflow.utils.task_group import TaskGroup
+from astro import sql as aql
+from astro.sql.table import Metadata, Table
+
+with DAG(
+    dag_id="advanced",
+    start_date=timezone.parse("2022-09-20"),
+) as dag:
+    sub_dir_1 = TaskGroup(group_id="sub_dir_1")
+    sub_dir_2 = TaskGroup(group_id="sub_dir_2")
+    sub_dir_1_1 = TaskGroup(group_id="sub_dir_1_1")
+    with sub_dir_1, sub_dir_1_1:
+        sub_dir_1_sub_dir_1_1_b = aql.transform_file(
+            file_path="""
+                select 1
+            """,
+            parameters={
+            },
+            op_kwargs={
+                "output_table": Table(
+                    name="b",
+                    metadata=Metadata(
+                        schema="sub_dir_1_1",
+                    ),
+                ),
+            },
+            task_id="b",
+        )
+    with sub_dir_1:
+        sub_dir_1_a = aql.transform_file(
+            file_path="""
+                select * from {{ sub_dir_1_sub_dir_1_1_b }}
+            """,
```

Comment:
@feluelle this isn't a file path, it's a SQL function, is this an error?

Reply:
Please read my https://github.com/astronomer/astro-sdk/pull/836#issuecomment-1256174948 above (with the note).

Reply:
fixed in 3372688